index.html

```diff
@@ -3,7 +3,435 @@
+              <button class="toggle" type="button" aria-label="icon">
+                <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false">
+                  <g>
+                    <circle cx="5" cy="12" r="2" class="toggle"></circle>
+                    <circle cx="12" cy="12" r="2" class="toggle"></circle>
+                    <circle cx="19" cy="12" r="2" class="toggle"></circle>
+                  </g>
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="sub-link">
+          <div class="link">
+            <a href="#" aria-label="channel logo">
+              <img class="img" alt="channel-logo" src="./image/freecodecampyoutube.jpg" />
+            </a>
+            <a href="#" class="channel-logo" aria-label="channel logo">freeCodeCamp.org</a>
+          </div>
+          <button class="subscribe" type="button" aria-label="icon">Subscribe</button>
+        </div>
+
+        <div>
+          <p class="show-more-text">
+            How do you contribute to open source? In this video, I'll describe the easiest way to
+            make a simple contribution to an open source project -- without using the command line,
+            a text editor, "forking a repo", or even touching Git outside of Github. I'll also
+            describe what kinds of contributions open source software projects are typically looking
+            for. As an example, I use submitting a student solution to a web development exercise
+            from The Odin Project, a free online curriculum for learning web development with Ruby
+            on Rails. You can find more information about that project at
+          </p>
+          <a href="#" target="_blank" aria-label="odin project website"
+            >http://theodinproject.com</a
+          >
+        </div>
+        <p class="show-more">Show more</p>
+
+        <section class="comment-section">
+          <header class="comment-head">
+            <p class="comment">2,493 Comments</p>
+            <p class="sort">SORT BY</p>
+          </header>
+          <div class="comments">
+            <img src="./image/abimbolayoutube.jpg" class="comment-img" alt="Avatar image" />
+            <p class="add-comment">Commenting publicly as Abimbola Buari</p>
+          </div>
+          <div class="other-comment">
+            <img class="comment-image" alt="avatar image" src="./image/commentyoutube.jpg" />
+            <div class="comment-block">
+              <h3>Nnechi<span> 4 year ago</span></h3>
+              <p>
+                I love this topic,Have been wanting to contribute to open source project for a long
+                to]ime but dont know how to go about it
+              </p>
+              <div class="comment-reply">
+                <p class="thousands">3K</p>
+                <p class="reply">REPLY</p>
+                <p>View 32 replies</p>
+              </div>
+            </div>
```

Comment:
There is an issue with this on how it renders, I think it's not arranged like others, see screenshot below:

![image](https://user-images.githubusercontent.com/36057474/95656144-34e30500-0b04-11eb-8236-1fa0bd72f163.png)

Reply:
This has been fixed, thanks.